api/src/pcapi/core/finance/api.py

```diff
@@ -532,15 +532,22 @@ def _generate_business_units_file() -> pathlib.Path:
     )
     row_formatter = lambda row: (
         human_ids.humanize(row.venue_id),
-        row.business_unit_siret,
-        row.business_unit_name,
-        row.venue_name,
+        _clean_trailing_newline(row.business_unit_siret),
+        _clean_trailing_newline(row.business_unit_name),
+        _clean_trailing_newline(row.venue_name),
         row.iban,
         row.bic,
     )
     return _write_csv("business_units", header, rows=query, row_formatter=row_formatter)
 
 
+def _clean_trailing_newline(row: str) -> str:
```

Comment:
Je trouve le nom `row` peu adapté. Ça n'est pas une _ligne_ (liste) de valeurs, mais une valeur unique. Appelons-la `value` ?

D'autre part, si je comprends bien ce que tu dis dans la description de la pull request, il est probable qu'on fasse évoluer cette fonction pour retirer d'autres caractères. On pourrait donc déjà lui donner un nom moins précis : 

```suggestion
def _clean_for_accounting(value: str) -> str:
```